src/Raven.Client/Documents/Operations/TimeSeries/GetTimeSeriesOperation.cs

```diff
@@ -13,44 +13,41 @@ public class GetTimeSeriesOperation : IOperation<TimeSeriesDetails>
     {
         private readonly string _docId;
         private readonly string _timeseries;
-        private readonly DateTime _from, _to;
+        private readonly List<(DateTime, DateTime)> _ranges;
 
         public GetTimeSeriesOperation(string docId, string timeseries, DateTime from, DateTime to)
         {
             _docId = docId;
             _timeseries = timeseries;
-            _from = from;
-            _to = to;
+            _ranges = new List<(DateTime, DateTime)>
+            {
+                (from, to)
+            };
+        }
+
+        public GetTimeSeriesOperation(string docId, string timeseries, List<(DateTime, DateTime)> ranges)
+        {
+            _docId = docId;
+            _timeseries = timeseries;
+            _ranges = ranges;
         }
 
         public RavenCommand<TimeSeriesDetails> GetCommand(IDocumentStore store, DocumentConventions conventions, JsonOperationContext context, HttpCache cache)
         {
-            return new GetTimeSeriesCommand(_docId, _timeseries, _from, _to);
+            return new GetTimeSeriesCommand(_docId, _timeseries, _ranges);
         }
 
         private class GetTimeSeriesCommand : RavenCommand<TimeSeriesDetails>
         {
             private readonly string _docId;
             private readonly string _timeseries;
-            private readonly DateTime _from, _to;
+            private readonly List<(DateTime From, DateTime To)> _ranges;
 
-            public GetTimeSeriesCommand(string docId, string timeseries, DateTime from, DateTime to)
+            public GetTimeSeriesCommand(string docId, string timeseries, List<(DateTime, DateTime)> ranges)
             {
-                _docId = docId;
-                _timeseries = timeseries;
-
-                if (from.Kind == DateTimeKind.Local)
```

Comment:
We don't need this code any more?

Reply:
moved it here: 
https://github.com/ravendb/ravendb/pull/9632/files/bad187f254c011a18a85cf7e276f8435c9488ac3#diff-8ccc7aca5fa95a155e4a440b9ad61d56R68